Epsilon.Host.Frontend/src/views/PerformanceDashboard.vue

```diff
@@ -35,20 +35,25 @@ import PersonalDevelopmentGraph from "@/components/PersonalDevelopmentGraph.vue"
 
 const data: Ref<CompetenceProfile | undefined> = ref(undefined)
 const App = new Api()
+const selectedTerm = ref<EnrollmentTerm | null>()
 
-const selectedTerm = ref<EnrollmentTerm | null>(null)
+function getTermDate(): string | null | undefined {
```

Comment:
I would prefer that this function has a parameter with a descriptive name to replace direct usage of the selectedTerm.value, this makes it a bit easier to understand.